Comp2023/src/main/java/frc/robot/commands/WristRunConstant.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.WRConsts;
+import frc.robot.subsystems.Wrist;
+
+/**
+ *
+ */
+public class WristRunConstant extends CommandBase
+{
+
+  private final Wrist   m_wrist;
+  private final boolean m_moveUp;
+
+  public WristRunConstant(Wrist wrist, boolean moveUp)
+  {
+    m_wrist = wrist;
+    m_moveUp = moveUp;
+
+    setName("WristRunConstant");
+    addRequirements(m_wrist);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize( )
+  {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute( )
+  {
+    m_wrist.moveWristConstantSpeed(WRConsts.kWristSpeedMaxManual * (m_moveUp ? 1 : -1));
```

Comment:
Remove the kWristSpeedManual constant, since it's getting scaled again in the moveWristConstantSpeed method by the same value.